src/app/pages/office/cat-b/office.cat-b.page.html

```diff
@@ -1,11 +1,205 @@
 <ion-header>
+    <practice-mode-banner *ngIf="isPracticeMode"></practice-mode-banner>
     <ion-toolbar>
-        <ion-title>office.cat-b</ion-title>
+        <ion-title>Office</ion-title>
     </ion-toolbar>
 </ion-header>
 
-<ion-content>
-    <ion-button (click)="navigateBack()">Back</ion-button>
-    <ion-button (click)="navigateForward()">To Rekey Reason</ion-button>
-    <ion-button (click)="navigateJournal()">To Journal</ion-button>
+<ion-content forceOverscroll="false">
```

Comment:
any page state variables within this page should be referencing ```pageState``` as the commonPageState should be merged into the pageState variable

Reply:
@matthew2564 - It works fine with pageState or commonPageState.  However it results in errors within the html for unresolved variables.  can we discuss b4 i change.